.github/workflows/call_black.yml

```diff
@@ -0,0 +1,15 @@
+# For more information see: https://black.readthedocs.io/en/stable/integrations/github_actions.html
+name: Lint
+
+on: 
+  # Allows you to run this workflow from another workflow
+  workflow_call
```

Comment:
oh I did not know this was doable: this should help with making reusable workflow component